pkgs/development/python-modules/pysim/default.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, pytestCheckHook
+, pytest-asyncio
+, colorlog
+, smpp_pdu
+, pyscard
+, packaging
+, gsm0338
+, bidict
+, jsonpath-ng
+, termcolor
+, pyyaml
+, pycryptodomex
+, construct
+, pyserial
+, pytlv
+, cmd2
+}:
+
+buildPythonPackage {
+  pname = "pysim";
+  version = "unstable-2023-08-13";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "osmocom";
+    repo = "pysim";
+    rev = "09ff0e2b433b7143d5b40b4494744569b805e554";
+    hash = "sha256-7IwIovGR0GcS1bidSqoytmombK6NkLSVAfKB2teW2JU=";
+  };
+
+  postPatch = ''
+    sed -i 's#smpp.pdu @ git+https://github.com/hologram-io/smpp.pdu#smpp.pdu#' setup.py
```

Comment:
small nitpick but we probably want `substituteInPlace` here since it's the default in nixpkgs

Reply:
Done.